rosidl_generator_dotnet/rosidl_generator_dotnet/__init__.py

```diff
@@ -159,3 +164,22 @@ def get_field_name(type_name, field_name):
         return "{0}_".format(type_name)
     else:
         return upperfirst(field_name)
+
+
+BASIC_DOTNET_TYPES_TO_BASIC_IDL_TYPES = {
```

Comment:
I can't find where this is used -- is it needed?

Reply:
Done